src/dissectors/ec_o5logon.c

```diff
@@ -46,6 +65,24 @@ void o5logon_init(void);
 
 /************************************************/
 
+#undef memrchr
+#define memrchr my_memrchr
```

Comment:
What is the purpose of this renaming? You shouldn't replace standard function names.
Why do you implement your own version of this standard function?